capstone-frontend/src/components/SearchBookModal.jsx

```diff
@@ -95,19 +85,10 @@ export class SearchBookModal extends Component {
 
         let updateJson;
 
-        if (this.props.type === 'club') {
-          updateJson = {
-            id: this.props.objectId,
-            gbookID: bookId,
-          }
-        }
-        else {
-          updateJson = {
-            id: this.props.objectId,
-            add_gbookIDs: bookId,
-          }
+        updateJson = {
```

Comment:
nit: This will be turned into JSON in the request body but is just a regular object right now, so naming it something like "updates" would be more accurate.

Reply:
Will fix